### ISSUE TITLE
RL: Agent, MultiAgent, EnvModel - Redefinitions for Hyperparameters

### ISSUE THREAD

**Issue by detlefarend** [fhswf/MLPro#277]
The methods for hyperparameter management are to be redefined because these classes have no own hyperparameters but their subcomponents have:

- [ ] **get_hyperparam_space** 
Returns a new hp space where all dimensions of the hp spaces of the underlying objects (e.g: Agent -> Policy, EnvModel) are added. 

- [ ] **get_hyperparam**
Returns a hp-tuple with all hp of the underlying objects.

Requires: #276

**Comment by detlefarend** [comment]
Hi Steve, this is not that easy and I think we have to extend the design of hyperparameters in general. That's why I currently labeled it as parked. A first idea is to create a new class HyperParamDispatcher(inherited from HyperParamTuple) that behaves like a tuple but internally it deals with a list of tuples. It has an additional method add_hp_tuple(). Something like that...
